.github/workflows/e2e-test-kind.yaml

```diff
@@ -90,9 +90,9 @@ jobs:
       - name: Install MinIO
         run:
           docker run -d --rm -p 9000:9000 -e "MINIO_ACCESS_KEY=minio" -e "MINIO_SECRET_KEY=minio123" -e "MINIO_DEFAULT_BUCKETS=bucket,additional-bucket" bitnami/minio:2021.6.17-debian-10-r7
-      - uses: engineerd/setup-kind@v0.5.0
+      - uses: engineerd/setup-kind@v0.6.2
         with:
-          version: "v0.21.0"
+          version: "v0.27.0"
```

Comment:
Please try to add `skipClusterLogsExport: true` in the `engineerd/setup-kind` action's `with` parameter to avoid error in the post step.

The new post step introduced in the `engineerd/setup-kind@v0.6.2` exports the action logs to a specified directory. That causes error due to running multiple actions, and all of them shares the same log directory, which is a UUID. 
As a result, only the first finished action can pass the post step. All the others fail due to the log directory already exists.

Reply:
i was toying with adding unique kind name... skip sounds easier.

Reply:
passing except lint (in CI).

Reply:
If we can have another way to fix the post-step issue, it would be great. If not, it's OK in my opinion.
There is another PR to address the linter issue.